packages/universal-components/src/FormFeedback/FormFeedbackTypes.js

```diff
@@ -2,7 +2,7 @@
 
 import * as React from 'react';
 
-type Type = 'help' | 'error';
+type Type = 'description' | 'error';
```

Comment:
Is this update necessary for payment form or is there any other reason for this change? For me help or hint is quite ok (I can imagine it will be used for something like `Min length 12 characters`).
Description is also ok, but slightly less clear for me. I see it on similar level as label or placeholder.

Reply:
Using help for text below the `TextInput` could be confusing if you want to use icon with help:
 
![Screenshot 2019-03-15 at 14 48 52](https://user-images.githubusercontent.com/858321/54435775-7d242080-4731-11e9-8655-5d332c9328be.png)

Do you have idea how to rename it? I didn't want to use something like `rightIcon`, because it's not clear what is the reason why icon is clickable.

Reply:
Oh, I see. Thanks. 
I think we can already use `suffix` for something like this, like in Orbit example: https://kiwicom.github.io/orbit-components/?selectedKind=InputField&selectedStory=With%20ButtonLink%20suffix&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
Good idea!

Reply:
Done